### ISSUE TITLE
XsectionDisplay Updates

### ISSUE THREAD

**Issue by AdamTheisen** [ARM-DOE/ACT#861]
The XSectionDisplay codes don't quite follow the same format as some of the other plotting functions.  The main issue is the dsname is the first variable.  We should update to follow similar formats as the other plotting code.
```   
    def plot_xsection_map(
        self, dsname, varname, subplot_index=(0,), coastlines=True, background=False, **kwargs
    ):  
```

**Comment by zssherman** [comment]
Makes sense to me!

**Comment by zssherman** [comment]
@AdamTheisen I noticed we are calling fields differently as well. We either use: data_field, varname, and field. Which would we want? We should have it the same across.

**Comment by zssherman** [comment]
Yeah I think field would be fine. We could open a poll as well.

**Comment by AdamTheisen** [comment]
We resolved this with merging in of #862
